src/components/application_manager/include/application_manager/hmi_language_handler.h

```diff
@@ -98,6 +98,13 @@ class HMILanguageHandler : public event_engine::EventObserver {
                                           hmi_apis::Common_Language::eType tts);
   void Init(resumption::LastState* value);
 
+  /**
+   * @brief Removes application from container after
+   * removing application from core
+   * @param app_id id application for removing
+  */
+  void OnUnregisterApplication(const uint32_t app_id);
```

Comment:
@pvvasilev , i took this code from branch release.4.1.0 . and it is bad idea to have the same functionality with different name in both branches.